utils/installer/install.sh

```diff
@@ -12,10 +12,12 @@ declare -xr XDG_DATA_HOME="${XDG_DATA_HOME:-"$HOME/.local/share"}"
 declare -xr XDG_CACHE_HOME="${XDG_CACHE_HOME:-"$HOME/.cache"}"
 declare -xr XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-"$HOME/.config"}"
 
+declare -xr NVIM_APPNAME="${NVIM_APPNAME:-"lvim"}"
+
 declare -xr LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$XDG_DATA_HOME/lunarvim"}"
```

Comment:
hmm good point about in general, but as I mentioned in the description, this was kept as-is to avoid having to migrate existing installations.

I'm not completely opposed to the idea if we find a different name than `core`, since that folder already exists.

> This way your data folder actually matches the `NVIM_APP` behavior and you have different data folders between different lvim installs

I'm guessing you mean by only changing `NVIM_APP`, right? because one can still change any of these variables independently.

Reply:
I might be missing something, but this shouldn't have an effect on existing installs anyway, after upgrading you'd still be using both the old paths and variables from the old install script.